plugin-management-library/src/test/resources/io/jenkins/tools/pluginmanager/impl/plugins.txt

```diff
@@ -46,7 +46,7 @@ role-strategy:2.16
 run-condition:1.3
 scm-api:2.6.3
 security-inspector:0.5
-snakeyaml-api:1.26.4
+snakeyaml-api:1.31-84.ve43da_fb_49d0b
```

Comment:
I think this data value can be retained to continue testing that behavior remains compatible even for older releases.
```suggestion
snakeyaml-api:1.26.4
```